<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chooseAlgo</class>
 <widget class="QWidget" name="chooseAlgo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>201</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;请选择调度算法&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_FCFS">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>先来先服务</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_SF">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>短作业优先</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_HPF">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>高优先权</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_RR">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>时间片轮转</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
